numpy/core/_ufunc_config.py

```diff
@@ -154,7 +154,7 @@ def geterr():
     --------
     >>> np.geterr()
     {'divide': 'warn', 'over': 'warn', 'under': 'ignore', 'invalid': 'warn'}
-    >>> np.arange(3.) / np.arange(3.)  # doctest: +SHOW_WARNINGS
```

Comment:
ahh, yes, this was a premature change that requires pytest-doctestplus (@seberg was originally working on the #23812 branch)

Reply:
@bsipocz Hmm, removing that fails differently on the refguide checks. I think the best short term fix is to skip that test. Could you make sure to add it back in your PR?